_includes/curriculum_listing.html

```diff
@@ -0,0 +1,52 @@
+{% assign curriculums = data.curriculum %}
+
+{% for curriculum in curriculums %}
+  <div class="row" id="{{curriculum.id}}">
+    <div class="col-sm-12">
+      <hr class="mt-5 mb-4">
+      <h3 class="h3 mb-4">{{curriculum.day}}: {{curriculum.topic}}</h3>
+    </div>
+    {% if curriculum.events %}
+      {% for event in curriculum.events %}
+        {% if event %} 
+        <div class="col-lg-12">
```

Comment:
card borders aren't closing on the right and bottom borders. another bootstrap issue that I believe may be related to the v4 version we are using?

Reply:
addressed in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/6bac5eed94e153bdb5256d50da58c8721eeaddd4